src/sql/platform/connection/common/constants.ts

```diff
@@ -11,6 +11,7 @@ export const outputChannelName = 'MSSQL';
 export const capabilitiesOptions = 'OPTIONS_METADATA';
 
 export const mssqlProviderName = 'MSSQL';
+export const pgsqlProviderName = 'PGSQL';
```

Comment:
why do we need pgsql hardcoded into the product?

Reply:
Originally, this was because MSSQL was also hardcoded into connectionWidget for looking up authType, and I added a case for PGSQL. I think I was able to remove the provider-specific lookup, though. Let me check if we need it. If not, I can remove it.

Reply:
Removed.